net/tsaddr/tsaddr_test.go

```diff
@@ -222,3 +222,97 @@ func TestContainsExitRoute(t *testing.T) {
 		}
 	}
 }
+
+func TestIsTailscaleIPv4(t *testing.T) {
+	type args struct {
+		ip netip.Addr
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "TestTailscaleIPv4",
+			args: args{
+				ip: netip.MustParseAddr("100.67.19.57"),
+			},
+			want: true,
+		},
+		{
+			name: "TestNotTailscaleIPv4",
```

Comment:
you're missing the case of an IPv4 in CGNAT but also in ChromeOS?

Reply:
Good catch, I will update my test.